apps/openssl.cnf

```diff
@@ -388,3 +388,11 @@ oldcert = $insta::certout # insta.cert.pem
 # Certificate revocation
 cmd = rr
 oldcert = $insta::certout # insta.cert.pem
+
+[pkcs12]
+certBagAttr = cb_attr
+
+# Uncomment this if you need Java compatible PKCS12 files
```

Comment:
Should this be commented out?

Reply:
@FireBurn My initial idea was to introduce this new config option commented out like this:
```
# Uncomment this if you need Java compatible PKCS12 files
[cb_attr]
#jdkTrustedKeyUsage = anyExtendedKeyUsage
```
So people who need it would uncomment it, and updating config wouldn't affect other users.

Reply:
yeah mistakenly left it in after previous change when testing. I'll update commit once I have a better handle on @mattcaswell 's comments

Reply:
I've compiled your tree locally, which allows me to create a pkcs12 truststore from pem files that works with Tomcat

I appreciate that the final code might be different, but this has proven really useful at my work

Reply:
@FireBurn excellent that's great to hear.